src/snowflake/connector/encryption_util.py

```diff
@@ -184,8 +169,8 @@ def encrypt_file(
     def decrypt_file(
         metadata,
         encryption_material,
-        in_filename,
-        chunk_size=block_size * 4 * 1024,
+        in_filename: str,
+        chunk_size: int = block_size * 4 * 1024,
```

Comment:
Why aren't you using the same formula to determine chunk size for both this decrypt method and the encrypt method above?

Reply:
This was left over from testing even bigger block sizes, I'll change this to match the other